Mathlib/Probability/Variance.lean

```diff
@@ -5,6 +5,7 @@ Authors: Sébastien Gouëzel, Kexing Ying
 -/
 import Mathlib.Probability.Notation
 import Mathlib.Probability.Integration
+import Mathlib.MeasureTheory.Measure.ProbabilityMeasure
```

Comment:
That new import is unused. Please remove it.
The build failed in CI. If you click on that, you will get to a page that tells you why (this unused import is the reason).

Reply:
Thank you for your detailed instruction. I removed the new import.